bgpd/bgp_aspath.h

```diff
@@ -67,6 +67,8 @@ struct aspath {
 
 /* `set as-path exclude ASn' */
 struct aspath_exclude {
+	struct aspath_exclude *next;
```

Comment:
I'm not terribly interested in having *another* list implementation added to the code base.  Please use an appropriate typesafe structure.  Until then this is a NACK from me.  We should be striving to remove the myriad special list implementations not add to them.

Reply:
taken into account